site/installing-scope.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Installing Weave Scope
+menu_order: 20
+---
+
+
+Weave Scope consists of three parts: the probe, the app and the user interface. It can be deployed either as a standalone configuration, or if you don’t want to manage the administration yourself, you can sign up for Weave Scope as a service.
+
+The following topics are discussed: 
+
+ * [Weave Scope Cloud Service](#signing-up-for-weave-scope-cloud-service)
+ * [Launching Scope as a Service with Docker Compose](#launching-scope-as-a-service-and-docker-compose)
+ * [Launching a Stack with Scope to AWS](#launching-a-stack-with-scope-to-aws)
+ * [Installing Scope as a DC/OS Package](#installing-scope-as-a-dc/os-package)
+ * [Visualizing Kubernetes Clusters with Scope](#visualizing-kubernetes-clusters-with-scope)
+
+
+##<a name="signing-up-for-weave-scope-cloud-service"></a>Signing Up for Weave Scope as a Service
+
+With Scope as a Service you can centrally manage and share access to your Scope user interface. In this configuration, the probe is run locally on your hosts, and the scope apps are hosted for you.
+
+To register with the service, go to [scope.weave.works](https://scope.weave.works), enter your email address and wait for the confirmation email that contains your service token.
+
+![`Weave Scope` Early Access Form](scope-early-access.png)
+
+
+>**Note**: Candidates for the Early Access Program are evaluated on a case per case basis. It may take a few days before you receive a confirmation email. To accelerate the process, please send an email to `help@weave.works` which explains your use-case.
+
+The service token is also accessible from the Scope as a Service main page after you've logged in:
+
+![Weave Scope Configure Your App](scope-cloud-main-page.png)
+
+
+###Launching Scope as a Service
+
+Run a probe on every machine that you want to monitor using the following:
+
+    sudo scope launch --service-token=<token>
+
+Where, 
+
+`--service-token=<token>`  is the token that was sent to you when you signed 
+up.
+
+Once the probes have been launched, log into the Scope service, and click on 'My Scope' in the top right-hand corner. 
+
+###Disabling Automatic Updates
```

Comment:
This section seems misplaced here (it is not service specific).  I'd stick it at the bottom of this doc for now.